### PR TITLE
feat(THEEDGE-3524): remove advisory column

### PR DESCRIPTION
**Pull request by acosferreira**
From cve detail, removed the advisoy colum list and filter.
Fixed the imageSetId Parameter
image

### CODE REVIEW

File: src/api/edge/imagesInfo.js [hunk 1]
```diff
@@ -5,7 +5,8 @@ export const useGetImageSet = () => {
   const axios = useAxiosWithPlatformInterceptors();
   const query = { limit: 10, offset: 0, sort_by: '-created_at' };
   const params = getTableParams(query);
-  return (id) => axios.get(`${EDGE_API}/image-sets/${id}?${params}`);
+  return (p) =>
```
Comment (bastilian) [inline src/api/edge/imagesInfo.js:8]: This is still getting passed an `id` or? 

In general we should avoid using one character variable names and try to be more descriptive.

File: src/components/VulnerabilityEdgePrompt.js [hunk 2]
```diff
@@ -46,7 +39,7 @@ const VulnerabilityEdgePrompt = () => {
       setImageSetID(device?.ImageInfo?.Image?.ImageSetID);
       const params = {
         id: 'view',
```
Comment (mkholjuraev) [inline src/components/VulnerabilityEdgePrompt.js:41]: Nitpick: this looks obsolete.

**Review by mkholjuraev — APPROVED** [review]
LGTM! Did not face issues while testing. As @acosferreira is on PTO, I will remove that line and then merge

File: src/components/VulnerabilityEdgePrompt.js (hunk 2 above)
Comment (mkholjuraev) [reply src/components/VulnerabilityEdgePrompt.js:41]: Weird, if I remove this line of code update page does not get opened. However, I am sure that this `id` is not used anywhere.

**Pull request merged**